fixed README.md for version 3.2 according to new rules

ficap opened the pull request.

Commit:
fixed README.md for version 3.2 according to new rules

hhorak commented:
One suggestion I have -- I think we should use RHEL-based image and also include link to the container catalog (it's possible to use direct links to the particular image), same as it is done in this MySQL's pull-request: https://github.com/sclorg/mysql-container/pull/188

Otherwise it looks good, please, create the same changes for 2.6, and 3.0-upg versions as well.

Commit:
added metadata for better manpage generating, example updated to rhscl based image, added link to Red Hat Container Catalog

Commit:
fixed README.md for version 2.6 according to new rules
added metadata for better manpages generating
updated example with rhscl based image
added link for Red Hat Container Catalog

Commit:
fix README.md for version 3.0-upg according to new rules
updated example with centos based image

omron93 commented:
@ficap Thank you. Merging.
